.gitignore

```diff
@@ -112,3 +112,5 @@ dropseq/13bd7eb0-23a3-4898-b043-f7e98223281g.v1/E-GEOD-81904.GSM2177575_Bipolar6
 10X/pbmc8k_S1_L008_I1_001.fastq.gz
 10X/pbmc8k_S1_L008_R1_001.fastq.gz
 10X/pbmc8k_S1_L008_R1_001.fastq.gz
+
+/tmp
```

Comment:
While doing development and wrangling data, I'm often downloading things, creating lists of things, and temporarily storing code that I don't want to check in.  Rails has a great pattern where every app has a tmp folder.  I'm propagating that pattern.